src/SAWScript/Interpreter.hs

```diff
@@ -1865,6 +1865,14 @@ primitives = Map.fromList
     , "name is used only for pretty-printing."
     ]
 
+  , prim "crucible_fresh_cryptol_var" "String -> Type -> CrucibleSetup Term"
+    (bicVal crucible_fresh_cryptol_var)
+    Experimental
+    [ "Create a fresh variable of the given Cryptol type for use within"
```

Comment:
```suggestion
    [ "Create a fresh symbolic variable of the given Cryptol type for use within"
```

Reply:
done in https://github.com/GaloisInc/saw-script/pull/759/commits/15908b0898278de5d3176826625d46058753139b